recipe/meta.yaml

```diff
@@ -22,16 +23,20 @@ requirements:
     - python
     - setuptools
     - wheel
-
   run:
-    - python
+    - python >=3.6
     - setuptools
     - pytest >=4.6
     - coverage >=5.2.1
     - toml
 
 test:
+  imports:
+    - pytest_cov
+  requires:
+    - pip
   commands:
+    - pip check
     - pytest --traceconfig | grep pytest-cov-{{ version }}
```

Comment:
Might have to add `python` and the proper pinnings on the `requires` section

Reply:
Done